draft-ietf-core-href.md

```diff
@@ -631,24 +631,29 @@ authority
 
 path
 : If the CRI reference contains a `discard` item of value `true`, the
-  path component is prefixed by a slash ("/") character.  If it
+  path component is considered *rooted*.  If it
   contains a `discard` item of value `0` and the `path` item is
   present, the conversion fails.  If it contains a positive discard
-  item, the path component is
+  item, the path component is considered *unrooted* and
   prefixed by as many "../" components as the `discard` value minus
   one indicates.
 
   If the discard item is not present and the CRI reference contains an
   authority that is `true`, the path component of the URI reference is
-  prefixed by the zero-length string.  Otherwise, the path component
-  is prefixed by a slash ("/") character.
-
-  If the CRI reference contains one or more `path` items, the prefix
-  is followed by the value of each item, separated by a slash ("/")
-  character.  <!-- A path segment that contains a colon character (e.g., -->
-  <!-- "this:that") cannot directly be used as the first such item.  Such a -->
-  <!-- segment MUST be preceded by a dot-segment (e.g., "./this:that") -->
-  <!-- unless scheme and/or authority are present. -->
+  considered unrooted.  Otherwise, the path component is considered
+  rooted.
+
+  If the CRI reference contains one or more `path` items, the path
+  component is constructed by concatenating the sequence of
+  representations of these items.  These representations generally
```

Comment:
```suggestion
  serializations of these items.  These serializations generally
```

Not perfectly happy with "serializations" either, but representations rings yet another bell in the REST context.

Reply:
Note from chat: this is still between information and data model, so not about wire format yet, so serialization isn't really making it better.